### ISSUE TITLE
Godot crashing when reimporting 3D models in a scene with an imported animation

### ISSUE THREAD

**Issue by Wondserek** [godotengine/godot#100484]
### Tested versions

v4.3.stable.official [77dcf97d8]

### System information

Godot v4.3.stable - Windows 10.0.19045 - GLES3 (Compatibility) - Intel(R) Iris(R) Xe Graphics (Intel Corporation; 27.20.100.8439) - 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz (6 Threads)

### Issue description

Basically what the title says. I'm experimenting with BlockBench animations and how to share them among other models, but whenever I have to reimport a model when the scene with it is open, the engine crashes. Here's what the console shows:

```
Godot Engine v4.3.stable.official.77dcf97d8 - https://godotengine.org
OpenGL API 3.3.0 - Build 27.20.100.8439 - Compatibility - Using Device: Intel - Intel(R) Iris(R) Xe Graphics

Editing project: C:/Art/Dev/Projects/guns-survivors-2d
Godot Engine v4.3.stable.official.77dcf97d8 - https://godotengine.org
OpenGL API 3.3.0 - Build 27.20.100.8439 - Compatibility - Using Device: Intel - Intel(R) Iris(R) Xe Graphics


================================================================
CrashHandlerException: Program crashed with signal 11
Engine version: Godot Engine v4.3.stable.official (77dcf97d82cbfe4e4615475fa52ca03da645dbd8)
Dumping the backtrace. Please include this when reporting the bug to the project developer.
[1] error(-1): no debug info in PE/COFF executable
[2] error(-1): no debug info in PE/COFF executable
[3] error(-1): no debug info in PE/COFF executable
[4] error(-1): no debug info in PE/COFF executable
[5] error(-1): no debug info in PE/COFF executable
[6] error(-1): no debug info in PE/COFF executable
[7] error(-1): no debug info in PE/COFF executable
[8] error(-1): no debug info in PE/COFF executable
[9] error(-1): no debug info in PE/COFF executable
[10] error(-1): no debug info in PE/COFF executable
[11] error(-1): no debug info in PE/COFF executable
[12] error(-1): no debug info in PE/COFF executable
[13] error(-1): no debug info in PE/COFF executable
[14] error(-1): no debug info in PE/COFF executable
[15] error(-1): no debug info in PE/COFF executable
[16] error(-1): no debug info in PE/COFF executable
[17] error(-1): no debug info in PE/COFF executable
[18] error(-1): no debug info in PE/COFF executable
[19] error(-1): no debug info in PE/COFF executable
[20] error(-1): no debug info in PE/COFF executable
[21] error(-1): no debug info in PE/COFF executable
[22] error(-1): no debug info in PE/COFF executable
[23] error(-1): no debug info in PE/COFF executable
[24] error(-1): no debug info in PE/COFF executable
[25] error(-1): no debug info in PE/COFF executable
[26] error(-1): no debug info in PE/COFF executable
[27] error(-1): no debug info in PE/COFF executable
[28] error(-1): no debug info in PE/COFF executable
[29] error(-1): no debug info in PE/COFF executable
[30] error(-1): no debug info in PE/COFF executable
[31] error(-1): no debug info in PE/COFF executable
[32] error(-1): no debug info in PE/COFF executable
[33] error(-1): no debug info in PE/COFF executable
[34] error(-1): no debug info in PE/COFF executable
[35] error(-1): no debug info in PE/COFF executable
[36] error(-1): no debug info in PE/COFF executable
[37] error(-1): no debug info in PE/COFF executable
[38] error(-1): no debug info in PE/COFF executable
[39] error(-1): no debug info in PE/COFF executable
[40] error(-1): no debug info in PE/COFF executable
[41] error(-1): no debug info in PE/COFF executable
-- END OF BACKTRACE --
================================================================
```

I'm searching for other posts about this for a while, but I still haven't found anything that could help me.

### Steps to reproduce

1. Open scene with an imported .gltf model with an animation
2. Open the "Animation" tab
3. Reimport the model

### Minimal reproduction project (MRP)

[minimal.zip](https://github.com/user-attachments/files/18155209/minimal.zip)

**Comment by fire** [comment]
@Calinou Do you know where your stable godot engine debug builds are so we can get precise crash traces?

**Comment by akien-mga** [comment]
> [@Calinou](https://github.com/Calinou) Do you know where your stable godot engine debug builds are so we can get precise crash traces?

https://github.com/Calinou/godot-debug-builds/releases/download/v4.3/godot-4.3-editor-debug-windows-msvc2022.zip

@Wondserek Could you test with the above build (4.3-stable Windows editor with debug symbols), which should give more details about the crash in the terminal?

**Comment by akien-mga** [comment]
Actually nevermind, this is a duplicate of #95769 which is fixed already in 4.4. The fix will be backported to 4.3.1.
